### DISCUSSION TITLE
How should CAM methods be evaluated

### DISCUSSION THREAD

**Discussion by noobgrow** [frgfm/torch-cam#122] [Q&A] [answered]
Sir, can you provide the other evaluation code like Average Drop/Incr. in Confidence/Insertion /Deletion ?Thanks a lot!

**Comment by frgfm** [comment]
Hi @noobgrow :wave: 

What do you mean? This repo is about producing class activations maps for a given trained model, there is no need for "evaluation" :thinking:

**Reply by frgfm** [reply]
Hi @noobgrow, would you mind sharing a bit more details please? :pray:

**Reply by noobgrow** [reply]
> evaluation

I mean that how to evaluate these methods which is better?

**Reply by frgfm** [reply]
Hello @noobgrow :wave:

You might be interested in #140 which will be closed quite soon as I opened a PR to add support for CAM method evaluation :)

**Comment by frgfm** [comment, marked as answer]
Hi everyone :wave: 

As you might know, #140 will add CAM method evaluation to the repo!
This commit will be staged for release 0.4.0 :+1:
